storage-engine/titan-configuration.md

```diff
@@ -167,7 +167,11 @@ To fully disable Titan for all existing and future data, you can follow these st
 
 3. After the compaction is finished, wait for the **Blob file count** metrics under **TiKV-Details**/**Titan - kv** to decrease to `0`.
 
-4. Update the configuration of these TiKV nodes to disable Titan.
+4. For TiDB v8.5.0 or a later version, update the configuration of these TiKV nodes to disable Titan.
+
+    > **Warning:**
+    >
+    > For versions earlier than v8.5.0, it is recommended to skip this step, because it might cause TiKV to crash. In these earlier versions, you can disable Titan directly by following Step 1. There is no performance difference between the configuration changes in Step 1 and the following changes in this step after the data migration is complete.
```

Comment:
```suggestion
    > For versions earlier than v8.5.0, it is recommended to skip this step, because it might cause TiKV to crash. In these earlier versions, following Step 1 is sufficient to disable Titan. There is no performance difference between the configuration changes in Step 1 and the following changes in this step after the data migration is complete.
```